File: class04/java/AP0409-ClassificaTriangulo.java
public class ClassificaTriangulo {
    public static void main(String[] x) {
        int a = 3, b = 3, c = 5;
        if (a == b && b == c) System.out.println("Equilatero");
        else if (a == b || b == c || a == c) System.out.println("Isosceles");
        else System.out.println("Escaleno");
    }
}

File: class04/cpp/AP0409-ClassificaTriangulo.cpp
#include <iostream>
int main(){int a=3,b=3,c=5;if(a==b&&b==c)std::cout<<"Equilatero\n";else if(a==b||b==c||a==c)std::cout<<"Isosceles\n";else std::cout<<"Escaleno\n";return 0;}
